packages/core/src/actions/smartAccount/buildUserOperationFromTxs.ts

```diff
@@ -77,8 +72,9 @@ export const buildUserOperationFromTxs: <
   };
   filterUndefined(_overrides);
 
-  return {
-    batch,
+  return buildUserOperation(client, {
+    uo: batch,
     overrides: _overrides,
-  };
```

Comment:
this is a breaking change....

Reply:
wasn't buildUserOperationFromTxs always returning uo struct with middleware pipeline run?
I just saw that now buildUserOperationFromTxs is just returning the uo batch for the txs, without calling buildUO function inside. so this change was intentional? I thought this was a mistake.

Reply:
https://github.com/alchemyplatform/aa-sdk/blob/v2.x.x/packages/core/src/provider/base.ts#L296 nope, it was always returning this object